test/example.md

```diff
@@ -64,6 +67,60 @@ methodName(4, '5', {},
 ```
 
 
+#### ExampleClass()
+
+
+ExampleClass description
+
+
+#### ExampleClass.method2(num)
+
+  - `num: `[`<number>`]
+
+method2 description
+
+
+#### ExampleClass.prototype.constructor(arg1, arg2)
+
+  - `arg1: `[`<Object>`]
+  - `arg2: `[`<string>`]
+
+ExampleClass constructor description
+
+
+#### ExampleClass.prototype.method1(arr)
+
+  - `arr: `[`<Array>`]
+
+method1 description
+
+
+#### PrototypeClass()
+
+
+PrototypeClass description and PrototypeClass
+
```

Comment:
Why was this empty line added?
What if I have
```
// PrototypeClass description and PrototypeClass
// and some more description
// or other text.
// Now constructor description
```
Will it separate after the first line?

Reply:
Yes, the first line is the title and everything else before parameters is a description, though in this case title should be fixed.